README.md

```diff
@@ -11,7 +11,9 @@ In `mix.exs`, add the ExMachina dependency:
 
 ```elixir
 def deps do
-  [{:ex_machina, "~> 0.4"}]
+  [
+    {:ex_machina, "~> 0.4", only: "test"}
+  ]
```

Comment:
My thinking was that I always copy-paste these, and it's more fiddly if it's condense. But if you still prefer that style, let me know and I'll change it.